strongbox-parent/pom.xml

```diff
@@ -63,7 +63,7 @@
         <version.spring>4.3.6.RELEASE</version.spring>
         <version.spring.security>4.2.1.RELEASE</version.spring.security>
         <version.spring.data>0.10.0-SNAPSHOT</version.spring.data>
-        <version.orientdb>2.2.16</version.orientdb>
+        <version.orientdb>2.2.18-SNAPSHOT</version.orientdb>
```

Comment:
@AlexOreshkevich : 

There was a bug in OrientDB `2.2.17` that we hit and @sbespalov reported it to the OrientDB team (for more details, check [here](https://github.com/orientechnologies/orientdb/issues/7172)).